app/controllers/miq_policy_controller/policy_profiles.rb

```diff
@@ -76,14 +75,12 @@ def profile_delete
     profiles = []
     # showing 1 policy set, delete it
     if params[:id].nil? || MiqPolicySet.find_by_id(params[:id]).nil?
-      add_flash(_("%{models} no longer exists") % {:models => ui_lookup(:model => "MiqPolicySet")},
-                :error)
+      add_flash(_("Policy Profile no longer exists"), :error)
     else
       profiles.push(params[:id])
     end
     process_profiles(profiles, "destroy") unless profiles.empty?
-    add_flash(_("The selected %{models} was deleted") %
-      {:models => ui_lookup(:models => "MiqPolicySet")}) if @flash_array.nil?
+    add_flash(_("The selected Policy Profile was deleted")) if @flash_array.nil?
```

Comment:
Deliberate plural -> singular correction.  The action is deleting 1 (currently selected) profile

(not seeing this flash in practice, seeing: »»Policy Profile« "foo": Delete successful«)